tools/ports/sdl2_image.py

```diff
@@ -88,7 +88,7 @@ def process_dependencies(settings):
     settings.USE_LIBJPEG = 1
 
 
-def handle_options(options):
+def handle_options(options, error_handler):
   opts['formats'].update({format.lower().strip() for format in options['formats'].split(',')})
```

Comment:
Maybe to a good time to add an error here if the format is not in the known list of formats?